interfaces/seccomp/backend.go

```diff
@@ -92,6 +102,7 @@ var globalProfileLE = []byte{
 	0x06, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0x7f,
 }
 
+// globalProfileBE is generated via cmd/snap-seccomp-blacklist
```

Comment:
Can you put a space to make the woke check happy before we address this properly in the follow-up?